src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.md

```diff
@@ -362,3 +362,50 @@ const foods = [
   ariaOptionChipRemovedText="removed"
 />;
 ```
+
+### Using functions for template literals
+
+```js
+import { MultiSelect } from 'suomifi-ui-components';
+
+const foods = [
+  {
+    labelText: 'Pizza',
+    uniqueItemId: 'pizza-123'
+  },
+  {
+    labelText: 'Burger',
+    uniqueItemId: 'burger-321'
+  }
+];
+
+const defaultSelectedTools = [
```

Comment:
Maybe `defaultSelectedFoods` would fit better for this constant.

Reply:
Renamed